### ISSUE TITLE
Open Dataset in Tableau: Login Required

### ISSUE THREAD

**Issue by EvanGranthamBrown** [ndoit/fenrir#181]
When the user is not logged in to Tableau Server and clicks the "TABLEAU" button on the dataset detail page, nothing happens; the workbook does not download. Need to implement a similar solution to the report page.

**Comment by RyanSnodgrass** [comment]
Possible duplicate of #133 ?

**Comment by EvanGranthamBrown** [comment]
Waiting on solution to the report/CAS auth issue.
